docs/understanding-tracking-design/managing-your-data-structures/cli/index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "Managing data structures via the CLI"
+description: "Use the 'snowplow-cli data-structures' command to manage your data structures."
+sidebar_label: "Using the CLI"
+sidebar_position: 2
+sidebar_custom_props:
+  offerings:
+    - bdp
+---
+
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+## Getting started
+
+The `data-structures` subcommand of [Snowplow CLI](https://github.com/snowplow-product/snowplow-cli) provides a collection of functionality to ease the integration of custom development and publishing workflows.
+
+### Snowplow CLI Prerequisites
+
+#### Download
```

Comment:
h4s render as simple text, so perhaps make them bold too? (not sure if that works)

Reply:
The styles changed under me :) (for the better) I'll try and make it look better.